assets/helpers/csrf/csrfReducer.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+
+// ----- Imports ----- //
+
+import type { Action } from './csrfActions';
+
+
+// ----- Types ----- //
+
+type Csrf = {
+    token: ?string,
+};
+
+
+// ----- Setup ----- //
+
+const initialState: Csrf = window.guardian && window.guardian.csrf ? {
```

Comment:
I didn't mean fix the user reducer in this PR, I meant fix them both in a new PR :-)

Maybe on Monday one of you can explain the `preloadedState` approach?